types/cesium/index.d.ts

```diff
@@ -4482,7 +4487,14 @@ declare namespace Cesium {
         width: number;
         loop: boolean;
         id: any;
-        constructor(options?: { show?: boolean; width?: number; loop?: boolean; material?: Material; positions?: Cartesian3[]; id?: any });
+        constructor(options?: { 
+            clampToGround?: Property;
```

Comment:
I don't believe `Polyline` has this constructor option: https://cesium.com/docs/cesiumjs-ref-doc/Polyline.html?classFilter=Polyline

Reply:
correct